.github/dependabot.yml

```
@@ -11,3 +11,12 @@ updates:
     commit-message:
       prefix: "chore"
       include: "scope"
+  - package-ecosystem: "npm"
+    directory: "/"
```

@teolemon should the directory be specified as this? Will Dependabot look for folders containing the npm package manifest?
```suggestion
    directory: "/taxonomy-editor-frontend"
```